picasso/src/test/java/com/squareup/picasso/BitmapHunterTest.java

```diff
@@ -405,6 +405,18 @@
     assertThat(shadowMatrix.getPreOperations()).containsOnly("rotate 90.0");
   }
 
+  @Test public void exifRotationSizing() throws Exception {
+    Request data = new Request.Builder(URI_1).rotate(-45).resize(5, 10).build();
```

Comment:
If I recall correctly the rotation value here affects the shadow matrix pre-operations that are tested in line 417.